source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py

```diff
@@ -1059,6 +1059,104 @@ def unproject_depth(depth: torch.Tensor, intrinsics: torch.Tensor) -> torch.Tens
     return points_xyz
 
 
+@torch.jit.script
+def convert_perspective_depth_image_to_orthogonal_depth_image(
+    perspective_depth: torch.Tensor, intrinsics: torch.Tensor
+) -> torch.Tensor:
+    """
+    Converts depth images captured by the distance_to_camera replicator
+    to depth images as if captured by the distance_to_image_plane replicator.
+
+    Provided a depth image where depth is provided as the distance to the principal
+    point of the camera (perspective depth), this function converts it so that depth
+    is provided as the distance to the camera's image plane (orthogonal depth).
+
+    This is helpful because `unproject_depth` assumes that depth is expressed in
+    the orthogonal depth format.
+
+    If `perspective_depth` is a batch of depth images and `intrinsics` is a single intrinsic matrix,
+    the same calibration matrix is applied to all depth images in the batch.
+
+    The function assumes that the width and height are both greater than 1.
+
+    Parameters:
+    -----------
+    perspective_depth : torch.Tensor
+        A tensor containing perspective depth images obtained with the distance_to_camera replicator.
+        The shape can be (H, W), (N, H, W), (H, W, 1), or (N, H, W, 1).
+
+    intrinsics : torch.Tensor
+        A tensor representing the intrinsics matrix of the standard camera.
+        The shape can be (3, 3) or (N, 3, 3).
+
+    Returns:
+    --------
+    torch.Tensor
+        A tensor containing depth images as if captured by the distance_to_image_plane replicator.
+        The shape will match the input `perspective_depth` shape.
+    """
```

Comment:
Can you add a `Raises:` section to indicate the exceptions raised and the cause?

E.g:
```
    Raises:
      ConnectionError: If no available port is found.
```